common/common-rest/src/main/java/io/servicecomb/common/rest/AbstractRestServer.java

```diff
@@ -1,228 +0,0 @@
-/*
- * Copyright 2017 Huawei Technologies Co., Ltd
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package io.servicecomb.common.rest;
-
-import java.nio.charset.StandardCharsets;
-import java.util.List;
-import java.util.Map;
-import java.util.Map.Entry;
-
-import javax.ws.rs.core.Response.Status;
-
-import org.apache.commons.lang3.StringUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import io.netty.buffer.Unpooled;
-import io.servicecomb.common.rest.codec.RestCodec;
-import io.servicecomb.common.rest.codec.produce.ProduceProcessor;
-import io.servicecomb.common.rest.codec.produce.ProduceProcessorManager;
-import io.servicecomb.common.rest.definition.RestOperationMeta;
-import io.servicecomb.common.rest.filter.HttpServerFilter;
-import io.servicecomb.common.rest.locator.OperationLocator;
-import io.servicecomb.common.rest.locator.ServicePathManager;
-import io.servicecomb.core.Const;
-import io.servicecomb.core.CseContext;
-import io.servicecomb.core.Invocation;
-import io.servicecomb.core.Transport;
-import io.servicecomb.core.definition.MicroserviceMeta;
-import io.servicecomb.core.definition.OperationMeta;
-import io.servicecomb.core.invocation.InvocationFactory;
-import io.servicecomb.foundation.common.utils.JsonUtils;
-import io.servicecomb.foundation.common.utils.SPIServiceUtils;
-import io.servicecomb.foundation.vertx.http.HttpServletRequestEx;
-import io.servicecomb.foundation.vertx.http.HttpServletResponseEx;
-import io.servicecomb.foundation.vertx.stream.BufferOutputStream;
-import io.servicecomb.serviceregistry.RegistryUtils;
-import io.servicecomb.swagger.invocation.Response;
-import io.servicecomb.swagger.invocation.exception.InvocationException;
-
-public class AbstractRestServer {
```

Comment:
Is it safe to remove the AbstractRestServer?

Reply:
this is our inner class, not a interface to developers